### PR TITLE
Fix/page email account migration

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

Replace email account migration wizard by a page

### CODE REVIEW

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 1]
```diff
@@ -319,32 +319,36 @@
...
+   <translation id="email_tab_modal_migrate_detination_title">Destination</translation>
+   <translation id="email_tab_modal_migrate_select_platform_text" qtlid="436658">Choississez le service de destination vers lequel votre adresse e-mail sera migré.</translation>
    <translation id="email_tab_modal_migrate_account_email_label" qtlid="436671">E-mail source :</translation>
    <translation id="email_tab_modal_migrate_service_type_label" qtlid="436684">Offre de destination :</translation>
    <translation id="email_tab_modal_migrate_destination_service_label" qtlid="436697">Service de destination :</translation>
-   <translation id="email_tab_modal_migrate_destination_service_help" qtlid="436710">Attention, vous ne pouvez migrer qu’un seul compte à la fois.</translation>
-   <translation id="email_tab_modal_migrate_password_text" qtlid="436723">Veuillez entrer le mot de passe de l’e-mail source pour valider la migration :</translation>
+   <translation id="email_tab_modal_migrate_destination_service_help">Il est possible de migrer un seul compte email à la fois.</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `e-mail`

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 2]
```diff
@@ -319,32 +319,36 @@
...
+   <translation id="email_tab_modal_migrate_password_text" qtlid="436723">Veuillez entrer le mot de passe de l’e-mail source pour valider la migration.</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: Please remove the `qtlid` if the translations should be changed as well

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 3]
```diff
@@ -319,32 +319,36 @@
...
    <translation id="email_tab_modal_migrate_no_remaining_available_emails" qtlid="436762">Après la migration, il ne vous restera plus de compte libre.</translation>
    <translation id="email_tab_modal_migrate_no_available_email" qtlid="436775">Vous n’avez plus de compte disponible sur votre service <strong>{0}</strong>. Choisissez un autre service dans la liste ou commandez un nouveau compte</translation>
    <translation id="email_tab_modal_migrate_order_email_service" qtlid="436788">Commander une solution e-mail</translation>
    <translation id="email_tab_modal_migrate_no_email_service" qtlid="436801">Vous n’avez pas de solution e-mail compatible pour la migration.</translation>
-   <translation id="email_tab_modal_migrate_confirmation_heading" qtlid="436814">Vous êtes sur le point de démarrer le processus de migration de votre adresse e-mail. Une fois cette action lancée, il ne vous sera plus possible :</translation>
+   <translation id="email_tab_modal_migrate_confirmation_before_continue"><strong>Avant de continuer...</strong></translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: Don't use tags in translations (even though you see tags everywhere)

Comment (FredericEspiau) [reply client/app/resources/i18n/email/Messages_fr_FR.xml]: Also, @antleblanc will ask to use …

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 4]
```diff
@@ -319,32 +319,36 @@
...
+   <translation id="email_tab_modal_migrate_confirmation_heading">Une fois la migration lancée, il ne vous sera plus possible :</translation>
    <translation id="email_tab_modal_migrate_confirmation_access_webmail" qtlid="444378">d’accéder à votre ancienne boîte e-mail;</translation>
    <translation id="email_tab_modal_migrate_confirmation_cancel_process" qtlid="444391">d'annuler le processus de migration.</translation>
-   <translation id="email_tab_modal_migrate_confirmation_delay" qtlid="436827">La migration du contenu peut prendre plusieurs heures selon la taille de la boite e-mail. <strong>L’action de migration est irréversible.</strong></translation>
+   <translation id="email_tab_modal_migrate_confirmation_delay">La migration du contenu peut prendre plusieurs heures selon la taille de la boite e-mail. <br /> Dès qu'elle sera terminée, votre boite email source sera supprimée dans un délai de 15 jours.</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: No tags in translations please

Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `e-mail`

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 5]
```diff
@@ -319,32 +319,36 @@
...
-   <translation id="email_tab_modal_migrate_configuration_guides_button" qtlid="436866">Cliquez ici pour accéder à nos guides techniques</translation>
+   <translation id="email_tab_modal_migrate_configuration_guides_button">Consulter nos guides techniques</translation>
    <translation id="email_tab_modal_migrate_aknowledge" qtlid="436879">En validant, je reconnais avoir pris connaissance de toutes les informations concernant la migration du compte e-mail source sélectionné.</translation>
-   <translation id="email_tab_modal_migrate_alert_not_supported" qtlid="436892">Attention, les éléments suivants ne seront pas pris en charge par la présente migration&#160;: <br />- vos filtres&#160;; <br />- vos répondeurs&#160;; <br />- vos mailing lists&#160;; <br />- vos contacts Roundcube.</translation>
+   <translation id="email_tab_modal_migrate_alert_not_supported">Les éléments suivants ne seront pas pris en charge par la migration&#160;: <br /><ul><li>vos filtres&#160;;</li><li>vos répondeurs&#160;;</li><li>vos mailing lists&#160;;</li><li>vos contacts Roundcube.</li></ul></translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: No tags in translations

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 6]
```diff
@@ -319,32 +319,36 @@
...
    <translation id="email_tab_modal_migrate_alert_configure_me" qtlid="436905">Il est nécessaire de disposer d’un compte e-mail non configuré (@configureme.me) pour réaliser l’opération.</translation>
-   <translation id="email_tab_modal_migrate_alert_dont_change_password" qtlid="444404">Attention, veuillez ne pas modifier le mot de passe de votre compte tant que sa migration n'est pas terminée.</translation>
-   <translation id="email_tab_modal_migrate_success" qtlid="436918">Votre compte e-mail est en cours de migration. L'opération sera effective d'ici 5 à 10 minutes.</translation>
+   <translation id="email_tab_modal_migrate_alert_dont_change_password"><strong>Veuillez ne pas modifier le mot de passe de votre compte email source tant que sa migration n'est pas terminée.</strong></translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: No tags in translations

Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `e-mail`

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 7]
```diff
@@ -319,32 +319,36 @@
...
+   <translation id="email_tab_modal_migrate_success">La migration de votre compte email vers votre service Email Pro ou Exchange est commencée. </translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `e-mail`

Comment (FredericEspiau) [reply client/app/resources/i18n/email/Messages_fr_FR.xml]: Can't you have 2 translations, one for each case ?

File: client/app/email-domain/email/email-domain-email.controller.js [hunk 8]
```diff
@@ -95,6 +96,14 @@ angular
                 });
         }
 
+        refreshSummary () {
+            this.Emails
```
Comment (FredericEspiau) [inline client/app/email-domain/email/email-domain-email.controller.js]: `return` promises please

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.controller.js [hunk 9]
```diff
@@ -106,48 +107,61 @@ angular.module("App").controller(
...
+
             this.Emails.migrateAccountToDestinationAccount(this.email.domain, this.email.accountName, this.migrate.destinationService, this.migrate.destinationEmail, this.migrate.password)
-                .then(() => this.Alerter.success(this.$scope.tr("email_tab_modal_migrate_success"), this.$scope.alerts.main))
+                .then(() => {
+                    this.Alerter.success(this.$scope.tr("email_tab_modal_migrate_success"), this.$scope.alerts.main);
+                    this.$scope.ctrlEmailDomainEmail.displayEmailsList();
+                })
                 .catch((err) => this.handleError(err))
-                .finally(() => this.$scope.resetAction());
+                .finally(() => { this.loaders.isWaitingForMigration = false; });
```
Comment (FredericEspiau) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.controller.js]: ESlint doesn't allow this formatting

File: client/app/email-domain/email-domain.service.js [hunk 10]
```diff
@@ -1091,9 +1091,23 @@ angular.module("services").service(
...
-            });
+            const defer = this.$q.defer();
+
+            this.OvhHttp
+                .get(`/email/domain/${domain}/account/${accountName}/migrate/${destinationServiceName}/destinationEmailAddress/${destinationEmailAddress}/checkMigrate`, {
+                    rootPath: "apiv6"
+                })
+                .then((data) => {
+                    // error codes are returned in success
+                    if (_.isArray(data.error) && data.error.length) {
```
Comment (FredericEspiau) [inline client/app/email-domain/email-domain.service.js]: `_.isArray(data.error) && !_.isEmpty(data.error)`

File: client/app/email-domain/email-domain.service.js [hunk 11]
```diff
@@ -1091,9 +1091,23 @@ angular.module("services").service(
...
+                        defer.reject(_.get(data, "error[0].code", ""));
```
Comment (FredericEspiau) [inline client/app/email-domain/email-domain.service.js]: You should handle all errors

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 12]
```diff
@@ -319,32 +319,58 @@
...
-   <translation id="email_tab_modal_migrate_success" qtlid="436918">Votre compte e-mail est en cours de migration. L'opération sera effective d'ici 5 à 10 minutes.</translation>
+   <translation id="email_tab_modal_migrate_alert_dont_change_password">Veuillez ne pas modifier le mot de passe de votre compte e-mail source tant que sa migration n'est pas terminée.</translation>
+   <translation id="email_tab_modal_migrate_success_emailpro">La migration de votre compte e-mail vers votre service Email Pro est commencée.</translation>
+   <translation id="email_tab_modal_migrate_success_exchange">La migration de votre compte e-mail vers votre service Exchange est commencée.</translation>
    <translation id="email_tab_modal_migrate_error" qtlid="436931">Une erreur s'est produite lors de la migration du compte :</translation>
+   <translation id="email_tab_modal_migrate_error_check_label" >Une erreur s'est produite lors de la migration du compte{0} :</translation>
+   <translation id="email_tab_modal_migrate_errors_check_label" >Plusieurs erreurs se sont produites lors de la migration du compte{0} :</translation>
```
Comment (antleblanc) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `" >` => `">`

Comment (antleblanc) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `" >` => `">`
`compte{0}` => `compte {0}` instead?

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 13]
```diff
@@ -319,32 +319,58 @@
...
-   <translation id="email_tab_modal_migrate_alert_dont_change_password" qtlid="444404">Attention, veuillez ne pas modifier le mot de passe de votre compte tant que sa migration n'est pas terminée.</translation>
-   <translation id="email_tab_modal_migrate_success" qtlid="436918">Votre compte e-mail est en cours de migration. L'opération sera effective d'ici 5 à 10 minutes.</translation>
+   <translation id="email_tab_modal_migrate_alert_dont_change_password">Veuillez ne pas modifier le mot de passe de votre compte e-mail source tant que sa migration n'est pas terminée.</translation>
+   <translation id="email_tab_modal_migrate_success_emailpro">La migration de votre compte e-mail vers votre service Email Pro est commencée.</translation>
+   <translation id="email_tab_modal_migrate_success_exchange">La migration de votre compte e-mail vers votre service Exchange est commencée.</translation>
    <translation id="email_tab_modal_migrate_error" qtlid="436931">Une erreur s'est produite lors de la migration du compte :</translation>
+   <translation id="email_tab_modal_migrate_error_check_label" >Une erreur s'est produite lors de la migration du compte{0} :</translation>
```
Comment (antleblanc) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `" >` => `">`
`compte{0}` => `compte {0}` instead?

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 14]
```diff
@@ -319,32 +319,58 @@
    <translation id="email_tab_modal_migrate_title" qtlid="436619">Migrer le compte</translation>
    <translation id="email_tab_modal_migrate_title_emailpro" qtlid="436632">Migrer vers E-mail Pro</translation>
    <translation id="email_tab_modal_migrate_title_exchange" qtlid="436645">Migrer vers Hosted Exchange</translation>
-   <translation id="email_tab_modal_migrate_select_platform_text" qtlid="436658">Choisissez le service de destination de votre adresse e-mail dans la liste ci-dessous.</translation>
+   <translation id="email_tab_modal_migrate_title_private_exchange" >Migrer vers Private Exchange</translation>
```
Comment (antleblanc) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: `" >` => `">`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 15]
```diff
@@ -1,21 +1,21 @@
-<div data-ng-controller="EmailsMigrateAccountCtrl as ctrl">
-    <div data-wizard
-        data-wizard-bread-crumb
-        data-wizard-on-cancel="resetAction"
-        data-wizard-on-finish="migrateAccount"
-        data-wizard-title="ctrl.wizardTitle">
+<div class="row" data-ng-controller="EmailsMigrateAccountCtrl as ctrl">
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: `.row > col-md-12` could be just a single `<div>` ;)

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 16]
```diff
@@ -26,202 +26,185 @@
...
+                        <span data-i18n-static="email_tab_modal_migrate_select_platform_text"></span>
+                        <span data-i18n-static="email_tab_modal_migrate_destination_service_help"></span>
                     </p>
-
+                    
+                    <!-- Not Supported Alert -->
+                    <div class="alert alert-info" role="alert">
+                        <p data-i18n-static="email_tab_modal_migrate_alert_not_supported"></p>
+                        <ul>
+                            <li data-i18n-static="email_tab_modal_migrate_alert_not_supported_filters "></li>
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: ` ">` => `">`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 17]
```diff
@@ -26,202 +26,185 @@
...
+                            <li data-i18n-static="email_tab_modal_migrate_alert_not_supported_responders "></li>
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: ` ">` => `">`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 18]
```diff
@@ -26,202 +26,185 @@
...
+                            <li data-i18n-static="email_tab_modal_migrate_alert_not_supported_lists"></li>
+                            <li data-i18n-static="email_tab_modal_migrate_alert_not_supported_roundcube_contacts"></li>
+                        </ul>
+                    </div>
+                    
+                    <div class="form-group">
+                        <label class="control-label" data-i18n-static="email_tab_modal_migrate_account_email_label"></label>
+                        <span class="d-block form-control-static" data-ng-bind="ctrl.email.email"></span>
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: `<span class="d-block form-control-static">` => `<p class="form-control-static">`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 19]
```diff
@@ -26,202 +26,185 @@
...
-                               data-i18n-static="email_tab_modal_migrate_service_type_label"></label>
-
+                        <label class="control-label" data-i18n-static="email_tab_modal_migrate_service_type_label"></label>
+                    
                         <!-- One Service -->
-                        <span class="d-block form-control-static"
-                              data-ng-if="ctrl.availableServices.length === 1"
-                              data-ng-bind="ctrl.migrate.serviceType"></span>
-
+                        <span class="d-block form-control-static" data-ng-if="ctrl.availableServices.length === 1" data-ng-bind="ctrl.migrate.serviceType"></span>
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: `<span class="d-block form-control-static">` => `<p class="form-control-static">`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 20]
```diff
@@ -26,202 +26,185 @@
...
 
-            <div class="text-center" data-ng-if="ctrl.loading">
-                <oui-spinner></oui-spinner>
+                <button type="submit" class="oui-button oui-button_primary" data-i18n-static="email_tab_modal_migrate_submit" ng-disabled="!ctrl.migrate.password || !ctrl.migrate.aknowledge || ctrl.loaders.isWaitingForMigration"></button>
+                <button class="oui-button oui-button_link" data-i18n-static="common_cancel" data-ng-click="ctrlEmailDomainEmail.displayEmailsList()"></button>
             </div>
+        </form>
+
+        <div class="text-center" data-ng-if="ctrl.loaders.isInitialRetrievalRunning || ctrl.loaders.isWaitingForDestinationEmails || ctrl.loaders.isWaitingForMigrationCheck || ctrl.loaders.isWaitingForMigration">
+            <oui-loader inline="true"></oui-loader>
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: ⚠️  `<oui-loader inline="true"></oui-loader>` => `<oui spinner></oui-spinner>`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 21]
```diff
@@ -26,202 +26,185 @@
...
+            data-ovh-alert-hide-remove-button>
+        </div>
+
+        
+        <div class="oui-message oui-message_warning" role="alert" ng-if="ctrl.shouldDisplayCheckMigrationsErrors">
+            <i class="oui-icon oui-icon-warning_circle oui-icon_bicolor" aria-hidden="true"></i>
+            <p class="oui-message__body">
+                <span data-ng-bind-html="ctrl.checkMigrationErrorLabel"></span>
+                <ul>
+                    <li ng-repeat="error in ctrl.checkMigrationErrors"
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: `ng-repeat` => `data-ng-repeat`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 22]
```diff
@@ -26,202 +26,185 @@
...
+        <div class="text-center" data-ng-if="ctrl.loaders.isInitialRetrievalRunning || ctrl.loaders.isWaitingForDestinationEmails || ctrl.loaders.isWaitingForMigrationCheck || ctrl.loaders.isWaitingForMigration">
+            <oui-loader inline="true"></oui-loader>
+        </div>
+        
+        <div data-ovh-alert="{{alerts.migrate}}"
+            data-ovh-alert-hide-remove-button>
+        </div>
+
+        
+        <div class="oui-message oui-message_warning" role="alert" ng-if="ctrl.shouldDisplayCheckMigrationsErrors">
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: `ng-if` => `data-ng-if`

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 23]
```diff
@@ -26,202 +26,185 @@
...
+            <p class="oui-message__body">
+                <span data-ng-bind-html="ctrl.checkMigrationErrorLabel"></span>
+                <ul>
+                    <li ng-repeat="error in ctrl.checkMigrationErrors" 
+                        data-ng-bind-html="error"></li>
+                </ul>
+            </p>
         </div>
+
+        <div ng-if="ctrl.shouldDisplayCheckMigrationsErrors">
```
Comment (antleblanc) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: `ng-if` => `data-ng-if`

File: client/app/email-domain/email-domain.service.js [hunk 24]
```diff
@@ -1091,13 +1091,27 @@ angular.module("services").service(
          * @param {string} destinationEmailAddress
          */
         checkMigrate (domain, accountName, destinationServiceName, destinationEmailAddress) {
-            return this.OvhHttp.get(`/email/domain/${domain}/account/${accountName}/migrate/${destinationServiceName}/destinationEmailAddress/${destinationEmailAddress}/checkMigrate`, {
-                rootPath: "apiv6"
-            });
+            const defer = this.$q.defer();
+
+            this.OvhHttp
+                .get(`/email/domain/${domain}/account/${accountName}/migrate/${destinationServiceName}/destinationEmailAddress/${destinationEmailAddress}/checkMigrate`, {
```
Comment (antleblanc) [inline client/app/email-domain/email-domain.service.js]: Do you have informations related to api endpoint status? (currently in beta)

File: client/app/resources/i18n/email/Messages_fr_FR.xml (hunk 12 above)
Comment (cbourgois) [reply client/app/resources/i18n/email/Messages_fr_FR.xml]: `compte{0}` : We concatenate with `email_tab_modal_migrate_error_check_should_retry` translation if we want to indicate to the user to wait before retry (optionnal, depends on error messages)

File: client/app/resources/i18n/email/Messages_fr_FR.xml (hunk 13 above)
Comment (cbourgois) [reply client/app/resources/i18n/email/Messages_fr_FR.xml]: `compte{0}` : Same here.

File: client/app/email-domain/email-domain.service.js [hunk 25]
```diff
@@ -1091,13 +1091,27 @@ angular.module("services").service(
          * @param {string} destinationEmailAddress
          */
         checkMigrate (domain, accountName, destinationServiceName, destinationEmailAddress) {
-            return this.OvhHttp.get(`/email/domain/${domain}/account/${accountName}/migrate/${destinationServiceName}/destinationEmailAddress/${destinationEmailAddress}/checkMigrate`, {
-                rootPath: "apiv6"
-            });
+            const defer = this.$q.defer();
```
Comment (jleveugle) [inline client/app/email-domain/email-domain.service.js]: We should avoid to do this. The API have to return an error state :)

Can we discuss about that when you're available ? :)

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 26]
```diff
@@ -319,32 +319,58 @@
...
-   <translation id="email_tab_modal_migrate_alert_dont_change_password" qtlid="444404">Attention, veuillez ne pas modifier le mot de passe de votre compte tant que sa migration n'est pas terminée.</translation>
-   <translation id="email_tab_modal_migrate_success" qtlid="436918">Votre compte e-mail est en cours de migration. L'opération sera effective d'ici 5 à 10 minutes.</translation>
+   <translation id="email_tab_modal_migrate_alert_dont_change_password">Veuillez ne pas modifier le mot de passe de votre compte e-mail source tant que sa migration n'est pas terminée.</translation>
+   <translation id="email_tab_modal_migrate_success_emailpro">La migration de votre compte e-mail vers votre service Email Pro est commencée.</translation>
+   <translation id="email_tab_modal_migrate_success_exchange">La migration de votre compte e-mail vers votre service Exchange est commencée.</translation>
    <translation id="email_tab_modal_migrate_error" qtlid="436931">Une erreur s'est produite lors de la migration du compte :</translation>
+   <translation id="email_tab_modal_migrate_error_check_label">Une erreur s'est produite lors de la migration du compte{0} :</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: Add a space between `compte` and `{0}`

File: client/app/resources/i18n/email/Messages_fr_FR.xml [hunk 27]
```diff
@@ -319,32 +319,58 @@
...
+   <translation id="email_tab_modal_migrate_errors_check_label">Plusieurs erreurs se sont produites lors de la migration du compte{0} :</translation>
```
Comment (FredericEspiau) [inline client/app/resources/i18n/email/Messages_fr_FR.xml]: Add a space between compte and {0}

File: client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html [hunk 28]
```diff
@@ -1,21 +1,19 @@
...
+            <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
+            <span data-i18n-static="common_back"></span>
+        </button>
+        <h2 data-i18n-static="email_tab_modal_migrate_title"></h2>
+
+        <form name="ctrl.migrateDestinationForm" novalidate data-ng-if="!ctrl.loaders.isInitialRetrievalRunning || ctrl.availableServices"
+            data-ng-submit="migrateAccount()">
+
+            <fieldset data-ng-disabled="ctrl.loaders.isWaitingForDestinationEmails || ctrl.loaders.isWaitingForMigrationCheck || ctrl.loaders.isWaitingForMigration">
+                <legend class="styleless mb-0" ><h3 class="header-underline" data-i18n-static="email_tab_modal_migrate_detination_title"></h3></legend>
```
Comment (FredericEspiau) [inline client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html]: There is a an extra space between

File: client/app/resources/i18n/email/Messages_fr_FR.xml (hunk 26 above)
Comment (cbourgois) [reply client/app/resources/i18n/email/Messages_fr_FR.xml]: `compte{0}` : We concatenate with `email_tab_modal_migrate_error_check_should_retry` translation if we want to indicate to the user to wait before retry (optionnal, depends on error messages)

File: client/app/resources/i18n/email/Messages_fr_FR.xml (hunk 27 above)
Comment (cbourgois) [reply client/app/resources/i18n/email/Messages_fr_FR.xml]: Same here

**Pull request merged**